### PR TITLE
This is a hot jar

### PR DESCRIPTION
**Pull request by DonSchado**
hello hotjar 🔥

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -477,6 +478,17 @@ def show
 end
 ```
 
+### Hotjar
+
+[Hotjar](https://www.hotjar.com/)
+
+```
+config.middleware.use(Rack::Tracker) do
+  handler :hotjar, { side_id: '1234' }
```
Comment (DonSchado) [inline README.md]: ha typo!!!

File: lib/rack/tracker.rb [hunk 2]
```diff
@@ -21,6 +21,7 @@
 require "rack/tracker/go_squared/go_squared"
 require "rack/tracker/criteo/criteo"
 require "rack/tracker/zanox/zanox"
+require "rack/tracker/hotjar/hotjar"
```
Comment (bumi) [inline lib/rack/tracker.rb]: is actually autoload still a thing?

File: README.md (hunk 1 above)
Comment (bumi) [reply README.md]: wanna change it?

File: lib/rack/tracker.rb (hunk 2 above)
Comment (DonSchado) [reply lib/rack/tracker.rb]: I think not

**Comment by bumi** [comment]
looks good! great refactoring also. next time Micha can implement his next tracking pixel :) 
(how do I approve my review?)

**Pull request merged**